tests/module/memory_test.c

```diff
@@ -0,0 +1,213 @@
+
+#include "redismodule.h"
```

Comment:
Why do you need a module to test this functionality? I would argue that redismodule.h should not even be in this library.